### PR TITLE
Split marathon-stats into marathon-stats-dev and marathon-stats-prod

### PR DESCRIPTION
**Pull request by xavi-**
This is the first infra-supported multi-app repo migration.

Context: https://clever.atlassian.net/browse/INFRA-1398

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -14,8 +14,11 @@ notify:
 publish:
   catapult:
     url: $$catapult_url
-    application: "marathon-stats"
+    applications:
+      - "marathon-stats-dev"
+      - "marathon-stats-prod"
   docker:
+    docker_version: 1.10.3
```
Comment (mohit) [inline .drone.yml:21]: this is fixed now right? we can just remove this?

**Comment by mohit** [comment]
other than the `docker_version` nit looks good to me.

File: launch/marathon-stats-prod.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+run:
```
Comment (mohit) [inline launch/marathon-stats-prod.yml:4]: maybe add a comment to why this is different from marathon-stats-dev?

File: .drone.yml [hunk 3]
```diff
@@ -14,8 +14,11 @@ notify:
 publish:
   catapult:
     url: $$catapult_url
-    application: "marathon-stats"
+    applications:
+      - "marathon-stats-dev"
```
Comment (nathanleiby) [inline .drone.yml:18]: nit: dev-infra ?

File: .drone.yml (hunk 1 above)
Comment (xavi-) [inline .drone.yml:21]: Just added it work around the docker issue we were seeing this morning.  Will remove.

File: launch/marathon-stats-prod.yml (hunk 2 above)
Comment (xavi-) [inline launch/marathon-stats-prod.yml:4]: Comments added in a4931a73f1888adb18d019368225e9792ca27cc9.

File: .drone.yml (hunk 3 above)
Comment (xavi-) [inline .drone.yml:18]: I still don't quite understand when to use dev-infra or dev.  I figured I should use `-dev` to match the name of the cluster it's getting stats from.

**Pull request merged**